Blog-projects/Widget-window-in-Qt-Quick-app/widgetFormHandler.cpp

```diff
@@ -0,0 +1,42 @@
+#include "widgetFormHandler.h"
+
+WidgetFormHandler::WidgetFormHandler(QObject *parent)
+    : QObject{parent}
+{
+    m_window = new FontControlsWidgetsForm();
+    m_window->setVisible(false);
+
+    QObject::connect(m_window, &FontControlsWidgetsForm::on_pushButton_clicked, this, &WidgetFormHandler::toggleWidgetsWindow);
+    QObject::connect(m_window, &FontControlsWidgetsForm::textChanged, this, &WidgetFormHandler::textChanged);
+    QObject::connect(m_window, &FontControlsWidgetsForm::fontChanged, this, &WidgetFormHandler::fontChanged);
+}
+
+bool WidgetFormHandler::isVisible()
+{
+    return m_window->isVisible();
+}
+
+void WidgetFormHandler::setVisible(bool visible)
+{
+    if (visible)
+        m_window->show();
+    else
+        m_window->hide();
+    emit visibleChanged();
+}
+
+QString WidgetFormHandler::getText()
```

Comment:
Getters should be const methods

Qt's best practice is to call such a getter `text()` rather than `getText()`, maybe we want to apply the same here?